app/src/main/java/it/quip/android/model/User.java

```diff
@@ -142,18 +110,20 @@ public static User getUserForSession() {
         }
 
         if (null == currentUser) {
-            currentUser = new User();
-            currentUser.setFacebookId(facebookId);
-            currentUser.saveInBackground();
+            try {
+                FacebookClient.createNewUser();
```

Comment:
Are we sure this isn't going to create duplicate users in the db? (duplicate meaning they have the same facebook id)